hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodeGeneratorMessageTaskFactoryProvider.java

```diff
@@ -82,7 +82,7 @@ public void init(ProcessingEnvironment env) {
 
     @Override
     public boolean process(Set<? extends TypeElement> elements, RoundEnvironment env) {
```

Comment:
I'm not that familair with this part of the code generator. But if I would be a third party developer and designing e.g. a Tree and want to expose it through a client. How can this be done? Because I see hard coded support for Enterprise/Jet/OS... but how can I get my tree in?